src/graphlib/graph.ts

```diff
@@ -24,7 +24,24 @@ var EDGE_KEY_DELIM = '\x01';
 //    edges up and, object properties, which have string keys, are the closest
 //    we're going to get to a performant hashtable in JavaScript.
 export class Graph {
-  constructor(opts = {}) {
+  _isDirected: boolean;
+  _isMultigraph: boolean;
+  _isCompound: boolean;
+  _label: any;
+  _defaultNodeLabelFn: () => any;
+  _defaultEdgeLabelFn: () => any;
+  _nodes: {};
+  _parent: {};
+  _children: {};
+  _in: {};
+  _preds: {};
+  _out: {};
+  _sucs: {};
+  _edgeObjs: {};
+  _edgeLabels: {};
```

Comment:
These properties are marked public. You can make them private by prepending them with the TypeScript `private` keyword, or you can use real JavaScript private properties